src/commonMain/kotlin/mechanics/Melee.kt

```diff
@@ -178,14 +182,40 @@ object Melee {
         return Random.nextDouble(min, max) + apToDamage(sp, totalAp, item, isNormalized)
     }
 
+    fun baseDamageRollPure(minDmg: Double, maxDmg: Double): Double {
```

Comment:
Possibly worth moving this to `General`, as it's not really related to anything melee-specific.  Very minor.